### PR TITLE
Sync up nginx.conf with latest changes

### PR DESCRIPTION
**Pull request by ihor-sviziev**
### Description
Sync nginx conf with latest changes in magento/magento2 repository.

https://github.com/magento/magento2/blob/069c55b6ba4c91662bbdcb31e77fe4a99e2800a2/nginx.conf.sample

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios
N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

**Comment by YPyltiai** [comment]
Thanks for contribution @ihor-sviziev !

@shiftedreality , @BaDos , @oshmyheliuk , please review.

File: nginx/1.9/etc/vhost.conf [hunk 1]
```diff
@@ -55,7 +60,7 @@ server {
             rewrite ^/static/(version\d*/)?(.*)$ /static/$2 last;
         }
 
-        location ~* \.(ico|jpg|jpeg|png|gif|svg|js|css|swf|eot|ttf|otf|woff|woff2)$ {
+        location ~* \.(ico|jpg|jpeg|png|gif|svg|js|css|swf|eot|ttf|otf|woff|woff2|json)$ {
```
Comment (oshmyheliuk) [inline nginx/1.9/etc/vhost.conf]: Don't understand why `json` type was added here, it's not a static file type that should be cached.

Comment (oshmyheliuk) [reply nginx/1.9/etc/vhost.conf]: I missed that this only for `.json` files in the `static` folder so it should be ok.

**Comment by andriyShevtsov** [comment]
QA approved

**Comment by shiftedreality** [comment]
@ihor-sviziev thank you for your contribution!

**Pull request merged**